src/pages/research_profile/ResearchProfile.tsx

```diff
@@ -187,6 +191,34 @@ const ResearchProfile = () => {
           </Box>
         )}
 
+        {(person.contactDetails?.email || person.contactDetails?.telephone || person.contactDetails?.webPage) && (
+          <Box sx={{ display: 'flex', flexDirection: 'column', gap: '0.25rem', mt: '0.75rem' }}>
```

Comment:
Hadde det vært en idè å lage to komponenter: `<ColumnFlexBox />` og `<RowFlexBox />`? Føler at disse går igjen i appen såpass mye at det blir overflødig å lage separate `styled`-komponenter for hver enkelt use-case 🤔

Reply:
`<Stack>`/`<Grid>` er jo egentlig MUI sin måte å gjøre dette på.
Bonus ved å bruke dem er at man får theme built-in, på en måte.
Personlig har jeg litt mixed emotions til både `Grid` og `Stack`, og bruker egentlig ikke noen av dem, fordi jeg føler at det blir en ekstra wrapper bare for å abstrahere bort CSS, og jeg frykter litt å miste kontrollen over hva man faktisk gjør. Ser `Grid` har blitt mye brukt i det siste, og det er helt greit, men syns samtidig det er viktig å være klar over at det aldri(?) blir en CSS-grid, men en CSS-flex (om den har prop `container`). Skjer noen sånne mystiske ting som jeg føler gjør at man mister litt kontrollen over hva man faktisk gjør (og hvorfor).

Det du foreslår har faktisk vært vedtatt en gang også, med `InputContainerBox` som ligger i `Wrappers.tsx`, uten at jeg syns det var noen suksess, egentlig 😅 Om du syns `<Stack>` er mer oversiktlig her er det helt legitimt å bruke det altså :)

Reply:
`Stack` har faktisk noen nice features også da, ser jeg, feks https://mui.com/material-ui/react-stack/#dividers